src/core/ConfigManager.cpp

```diff
@@ -699,12 +687,12 @@ void ConfigManager::initDevelopmentWorkingDir()
 				done++;
 			}
 			if (line.startsWith("lmms_BINARY_DIR:")) {
-				m_lmmsRcFile = line.section('=', -1).trimmed() +  QDir::separator() +
-							   ".lmmsrc.xml";
+				m_lmmsRcFile = line.section('=', -1).trimmed() +  QDir::separator() + ".lmmsrc.xml";
 				done++;
 			}
 			if (done == 2)
 			{
+				m_isDevelopment = true;
```

Comment:
This will only set `m_isDevelopment` if there are further lines after the two that this code looks for. I expect that this will usually be the case, but we shouldn't rely on it. I would instead set it in the sme block as `m_lmmsRcFile`, as that's all we care about for the config manager. The data directory code is irrelevant to this class, and only here so we don't need to duplicate the CMake cache scanning elsewhere.